contracts/staking/tests/ValidatorManagerTests.t.sol

```diff
@@ -127,7 +127,7 @@ abstract contract ValidatorManagerTest is Test {
         });
     }
 
-    function testInitializeEndValidation() public {
+    function testInitializeEndValidation() public virtual {
```

Comment:
Made virtual so it can be overridden in the PoS test file, in order to induce uptime logic.